fem/fespace.hpp

```diff
@@ -947,12 +947,10 @@ class L2ProjectionGridTransfer : public GridTransfer
       const FiniteElementSpace &fes_ho;
       const FiniteElementSpace &fes_lor;
 
-      int ndof_lor, ndof_ho, nref;
+      mutable Array<double> R, P;
+      Array<int> offsets;
```

Comment:
I realize this is private, but I think brief documentation of `offsets` would make the code easier to maintain.

Reply:
Good idea, I added a comment.